js/ui/html_editor/utils/image_uploader_helper.js

```diff
@@ -79,76 +101,66 @@ export class ImageUploader {
         };
     }
 
-    createTabs(formData) {
-        const result = [];
-
-        if(!this.actualTabs || this.isUpdating) {
-            this.actualTabs = ['url'];
-        }
+    createUrlTab(formData) {
+        return new UrlTab(this.module, {
+            config: this.config,
+            formData,
+            isUpdating: this.isUpdating
+        }, this.updateAddButtonState.bind(this));
+    }
 
-        this.actualTabs = this.normalizeTabs(this.actualTabs);
-
-        this.actualTabs.forEach((tabName) => {
-            const newTab = tabName === 'url'
-                ? new UrlTab(this.module, {
-                    config: this.config,
-                    formData,
-                    isUpdating: this.isUpdating
-                })
-                : new FileTab(this.module, {
-                    config: this.config
-                });
+    createFileTab() {
+        return new FileTab(this.module, {
+            config: this.config
+        }, this.updateAddButtonState.bind(this));
```

Comment:
Minor comment.

Maybe, should we change the  ```this.updateAddButtonState``` method to the arrow function to forget about the bind?